barretenberg/cpp/src/barretenberg/vm/avm/generated/flavor.cpp

```diff
@@ -2207,6 +2207,10 @@ AvmFlavor::CommitmentLabels::CommitmentLabels()
     Base::incl_mem_tag_err_counts = "INCL_MEM_TAG_ERR_COUNTS";
 };
 
+// Note: current de-/serialization routines are not including the padded zero univariates which are added as part of
```

Comment:
AFAIK, this code is not used and therefore I did not want to adapt some code which we might remove. There are other flavors for which these serialization routines are used but not ours. Since I was not sure if there might be used in the future I did not delete this code.